adops/setting-up-prebid-video-in-dfp.md

```diff
@@ -1,10 +1,96 @@
 ---
 layout: page_v2
-title: Setting up Prebid Video in Google Ad Manager
-head_title: Setting up Prebid Video in Google Ad Manager
-description: Setting up Prebid Video in Google Ad Manager
+title: GAM Step by Step - Video Creatives
+head_title: GAM Step by Step - Video Creatives
+description: Setting up in-player and long-form video for Prebid in Google Ad Manager
 top_nav_section: adops
 sidebarType: 3
 ---
 
-{% include adops/adops-gam-video-setup.html %}
+# GAM Step by Step - Video Creatives
+{: .no_toc }
+
+* TOC
+{:toc}
+
+This page walks you through the steps required to create in-player and long-form video creatives to attach to your Prebid line items in Google Ad Manager (GAM).
+
+{: .alert.alert-success :}
+For complete instructions on setting up Prebid line items in Google Ad Manager, see [Google Ad Manager with Prebid Step by Step](/adops/step-by-step.html).
+
+{: .alert.alert-info :}
+For engineering setup instructions, see [Show Video Ads with a Google Ad Manager Video Tag](/dev-docs/show-video-with-a-dfp-video-tag.html).
+
+
+Each VAST creative contains a URL that points to the cached VAST XML. (This is because most video players can only work with a URL that returns VAST XML.) When setting up video creatives, it's important to understand where the VAST XML is stored for each of your bidders. The most common place to store VAST XML is the AppNexus cache, but some bidders (such as RubiconProject and SpotX) use their own cache services. To support such bidders, see [Multiple Cache Locations](#multiple-cache-locations) below.
+
+## Single Cache Location
+
+If you only use bidders that provide full VAST responses, do the following:
```

Comment:
Please change to "All of your bidders may use the same VAST cache server in these scenarios:

- mobile SDK video interacts with Prebid Server which will be set up to cache VAST
- if all of your Prebid.js bidders return VAST XML
- if you utilize the Prebid.js [ignoreBidderCacheKey](/dev-docs/publisher-api-reference/setConfig.html#setConfig-vast-cache) option

In any of these cases, you only need to set up creatives pointing to the VAST cache: